PUBLISHING.md

```diff
@@ -0,0 +1,46 @@
+## Publishing a new version
+
+To publish the npm package, you'll need an [npm](https://www.npmjs.com/) account, and you'll need to be added to the list of maintainers by one of the current RoverUI maintainers.
+
+### With write access
+
+1. Checkout a release branch of some kind: (e.g. `release-2.0.4` or `new-v2.1.4`)
+2. **Following [semver](https://semver.org)** as your version guide, use the script `yarn new-version [value]` to bump the version. See [`yarn version`](https://classic.yarnpkg.com/en/docs/cli/version/) docs for more info
+   1. For `patch`: `yarn new-version --patch`
+   2. For `minor`: `yarn new-version --minor`
+   3. For `major`: `yarn new-version --major`
+   4. For other: `yarn new-version --new-version 3.4.11`
+3. Open a PR for that branch
+4. Once you have at least one PR approval from an admin and no pending requests for changes, you can merge in to master.
+5. After your pull request is merged, follow this process:
+   1. `git checkout master && git fetch -p && git reset --hard origin/master`
+   2. `yarn publish`
+6. Ask an admin to push your new version's tag to GitHub.
+
+### With admin access
+
+If you're an [admin](https://github.com/cision/rover-ui/settings/access), you can commit directly to `master`. The process is much more streamlined.
```

Comment:
I replaced references to champions with the more cut-and-dry "admins", since anyone with admin rights is welcome to do this.